pteranodon/plugins/ext_plugins/power/power.py

```diff
@@ -0,0 +1,128 @@
+import time
+from asyncio import AbstractEventLoop
+from logging import Logger
+from typing import Dict, Optional
+from collections import deque
+from subprocess import SubprocessError
+
+import numpy as np
+from mavsdk import System, telemetry
+
+from pteranodon.plugins.ext_plugins.abstract_custom_plugin import AbstractCustomPlugin
+from .tegrastats import Tegrastats
+
+
+class Power(AbstractCustomPlugin):
+    """
+    Generates battery usage information
+    """
+
+    def __init__(
+        self,
+        system: System,
+        loop: AbstractEventLoop,
+        logger: Logger,
+        base_plugins: Dict,
+        ext_args: Dict,
+    ) -> None:
+        super().__init__("power", system, loop, logger, base_plugins, ext_args)
+        self._telemetry = self._base_plugins["telemetry"]
+        self._param = self._base_plugins["param"]
+        self._capacity = self._param.get_param_float("BAT1_CAPACITY")
+        self._num_cells = self._param.get_param_int("BAT1_N_CELLS")
+        if self._ext_args["power"] is not None:
```

Comment:
Assign arguments to variables and then can remove this duplicated code in if statement